services/user.py

```diff
@@ -0,0 +1,36 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import User
+
+def create_user(username: str, password: str, email: str = None, first_name: str = None, last_name: str = None) -> User:
+    user = get_user_model().objects.create_user(username=username, password=password)
+
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+
+    user.set_password(password)
+    user.save()

```

Comment:
According to the checklist, you should call the `save()` method only one time if it's possible. You're calling `save()` here and also on line 36. Try to refactor your code to call `save()` only once.